shell/platform/embedder/embedder.h

```diff
@@ -306,12 +306,26 @@ typedef bool (*TextureFrameCallback)(void* /* user data */,
                                      FlutterOpenGLTexture* /* texture out */);
 typedef void (*VsyncCallback)(void* /* user data */, intptr_t /* baton */);
 
+typedef struct {
```

Comment:
This struct should start with a size in case we want to add more information to it in the future.

Reply:
Done